src/App.css

```diff
@@ -0,0 +1,232 @@
+.App {
+  text-align: center;
+}
+
+.App-logo {
+  height: 40vmin;
+  pointer-events: none;
+}
+
+@media (prefers-reduced-motion: no-preference) {
+  .App-logo {
+    animation: App-logo-spin infinite 20s linear;
+  }
+}
+
+body {
+  background-color: #c4bcbc;
+}
+
+.border {
+  border-radius: 30px;
+}
+
+.chan {
```

Comment:
what is chan ?

Reply:
I forget to delete this file .This file is not referenced by my code.